Rick-and-Morty/Rick And Morty/Views/CharacterCard.swift

```diff
@@ -0,0 +1,92 @@
+                    Text(characterCardState.name)
+                        .font(.title)
+                        .fontWeight(.black)
+                    HStack {
+                        Circle()
+                            .foregroundColor(characterCardState.isAlive ? .green : .red)
+                            .frame(width: 10, height: 10)
+                        Text(characterCardState.isAlive ? "Alive - \(characterCardState.species)" : "Dead - \(characterCardState.species)")
+                    }
+                }
+                
+                VStack(alignment: .leading, spacing: noSpacing) {
+                    Text("Last known location:")
+                        .font(.caption)
+                        .foregroundColor(.secondary)
+                    Text(characterCardState.lastLocation)
+                }
+                
+                VStack(alignment: .leading, spacing: noSpacing) {
+                    Text("First seen in:")
+                        .font(.caption)
+                        .foregroundColor(.secondary)
+                    Text(characterCardState.firstEpisode)
+                }
+            }
+            Spacer()
+        }
+        .cornerRadius(cornerRadius)
+        .overlay(
+            RoundedRectangle(cornerRadius: cornerRadius)
+                .stroke(Color(.lightGray), lineWidth: 0.25)
+        )
+        .padding()
+        .frame(height: 200)
+    }
+}
+
+struct CharacterTileView_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            VStack {
+                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: true, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: false, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: true, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+            }
+            .preferredColorScheme(.light)
+            VStack {
+                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: true, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: false, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: true, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+            }
+            .preferredColorScheme(.dark)
```

Comment:
Theres duplication here, try and use my above example to see what functions you can create for the VStack and CharacterCard duplications :)